tools/prow-auto-bumper/main.go

```diff
@@ -65,6 +65,11 @@ var (
 	targetTime = time.Now().Add(-time.Hour * 7 * 24) // 7 days
 )
 
+// Client handles methods for github issues
+type Client struct {
```

Comment:
I think you'll add more clients here, so better use plural.

Reply:
renamed GHClientWrapper